### DISCUSSION TITLE
Import scan results

### DISCUSSION THREAD

**Discussion by Ghoatmountain** [qarmin/czkawka#1128] [Q&A]
Hi,
is there a way to import saved scan results from json to suite?

**Comment by qarmin** [comment]
Not yet, there is already issue about it, but I doubt that I will do this for old gui

**Reply by clarenceHub** [reply]
forgive my ignorance but I'm wondering if that "save scan" idea will help sove my issue. I have a comparison folder of "unwanted" (but necessary for this program) files is nearing 1tb, so was wondering if this solution (of saving past file hashes/scan results) will allow me to delete the "unwanted" files that I currently only keep for this program.

I download pics and vids but the site I use often has repeats uploaded at a later date. I keep all unwanted files (that I would usually delete) in a separate folder for this program to compare and omit these newer downloads that are repeats. Naturally, it also removes dupes that I have stored in another folder that are keepers. I scan a set of folders and choose "Select all except oldest". This deletes the newly downloaded files that are dupes but keeps the old ones to compare to when I download new files.

A "save scan/file hashes" of a specific folder and not all folders would be ideal alongside easy 1-button backup as this folder is always being updated.
